server/services/taskService.ts

```diff
@@ -65,10 +66,23 @@ export default class TaskService {
     return results
   }
 
-  async find(token: string, premisesId: string, taskType: string): Promise<TaskWrapper> {
+  async find(
+    token: string,
+    premisesId: string,
+    taskType: string,
+    userFilters?: { apAreaId?: string; qualification?: UserQualification },
+  ): Promise<TaskWrapper> {
     const taskClient = this.taskClientFactory(token)
 
     const task = await taskClient.find(premisesId, taskType)
+    if (userFilters) {
```

Comment:
This will always be true if `userFilters` is an object as even an empty object is truthy.

Reply:
Ah but it will not be falsy if no argument provided? As it is not a required parameter

Reply:
It would be if but I *think* the only place we're calling it is in the tasksController show method where we're passing in an object. The object properties could be undefined but the object itself will always be defined. Ignore me if we're calling this somewhere else or if I missed something!

Reply:
Ah but it will not be falsy if no argument provided? As it is not a required parameter

If I remove the if statement, I will always need to provide an object

Reply:
yea - fair enough! Shall amend to make it required :)

Reply:
have kept an if statement for each of the properties to prevent an unnecessary loop - what do you think?

Reply:
yeah that's a good idea